json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -137,6 +138,12 @@ public function get_settings_response() {
 			case 'URL' :
 				$response[$key] = (string) home_url();
 				break;
+			case 'locale_variant':
+				$blog_locale_variant = wpcom_l10n_get_blog_locale_variant();
```

Comment:
Should be preceded with `defined ( 'IS_WPCOM' ) && IS_WPCOM`. I don't think we need a value otherwise